src/pip-delete-this-directory.txt

```diff
@@ -0,0 +1,5 @@
+This file is placed here by pip to indicate the source was put
```

Comment:
It needs to be part of the pull request, this might help: http://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste